CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 lib_sw_pll library change log
 =============================
 
+UNRELEASED
+----------
+
+  * ADDED:     Support for XCommon CMake build system
```

Comment:
I've just bumped to 2.0.1 (patch increment) since I made a fix.
This is a new feature so bumping to 2.1.0 makes sense.
I like the UNRELEASE tag in principle but it might cause issues when new features are on develop but the version number is the same as the last release tag.